Policies/github-terms-of-service.md

```diff
@@ -2231,38 +2231,42 @@ We will have no liability for any failure or delay due to matters beyond our rea
 
 <TABLE>
   <TR><TD ALIGN="LEFT">
-&dagger; These Terms of Service, together with the GitHub Privacy Statement,
+&dagger;&nbsp;These&nbsp;Terms&nbsp;of&nbsp;Service,&nbsp;together&nbsp;with&nbsp;the&nbsp;GitHub&nbsp;Privacy&nbsp;Statement,
 </TD></TR>
   <!-- I don't understand it, but somehow setting the width to just 1%, seems to make the width be 100%
        such that all the space in the width is used --> 
-  <TR><TD ALIGN="RIGHT" WIDTH="1%">
+  <TR><TD ALIGN="RIGHT" WIDTH="1%" COLSPAN="2">
 represent the complete and exclusive statement of the agreement between you and us.
```

Comment:
```suggestion
represent the complete and exclusive statement of the agreement between you and us.
```

Reply:
Hello @Lovegill5, thanks for approving this pull request.

Slightly confused about your suggested changes because they don't seem to do anything. Are they mistakes?

BTW, I've made further changes since the last PR snapshot, and plan to make even more changes. Going through an extensive drafting process.

Thanks.